update pass fail criteria and editorial fixes (#47)
biometricITC/cPP-toolboxes: toolbox.adoc

|===

As explained in <<BIOPP-Module>>, presentation attacks can be done by attempts or transactions.

> The maximum number of attempts/transactions allowed with one artefact is twenty (20). If three (3) matches are made to the artefact, the independent test fails (further attempts/transactions are not necessary even if 20 total attempts/transactions have not yet been made) because IAPAR exceeds 15% that is allowable maximum value specified in FIA_MBV_EXT.3.1..

Reviewer: ```suggestion
The maximum number of attempts/transactions allowed with one artefact is twenty (20). If three (3) matches are made to the artefact, the independent test fails (further attempts/transactions are not necessary even if 20 total attempts/transactions have not yet been made) because the IAPAR has exceeded 15%, the allowable maximum value specified in FIA_MBV_EXT.3.1.
```

== Guidance for Penetration Testing (AVA_VAN.1)

> The evaluator moves to penetration testing only if the TOE passes independent testing. As described in <<BIOSD>> Section 7, the evaluator shall select those artefacts that show higher IAPAR during independent testing or higher quality artefacts.

Reviewer: ```suggestion
The evaluator moves to penetration testing only if the TOE passes independent testing. As described in <<BIOSD>> Section 7, the evaluator shall select those artefacts that show a higher IAPAR during independent testing or higher quality artefacts.
```

* Each artefact shall be presented to the TOE an arbitrary number of times within allowed time period. As described in <<BIOSD>>, both independent and penetration testing shall be finished within one week.

=== Number of Subjects
If the evaluator can create artefacts that produce an attack presentation match during independent testing, the evaluator should select the test subjects whose artefacts had successful matches and increase the number of attempts/transactions. The evaluator may replace the test subject for penetration testing as described in <<BIOSD>> Section 7.

=== Pass/Fail Criteria

> As described in <<BIOSD>>, both independent and penetration testing shall be finished within one week. The evaluator may select one or two artefacts and perform an arbitrary number of attempts/transactions within this time period. If the evaluator can create artefacts that can reproducibly overcome the TOE at the higher IAPAR specified in FIA_MBV_EXT.3.1, the TOE fails AVA_VAN.1 evaluation.

Reviewer: ```suggestion
As described in <<BIOSD>>, both independent and penetration testing shall be finished within one week. The evaluator may select one or two artefacts and perform an arbitrary number of attempts/transactions within this time period. If the evaluator can create artefacts that reproducibly cause the TOE to achieve an IAPAR higher than what is specified in FIA_MBV_EXT.3.1, the TOE fails AVA_VAN.1 evaluation.
```
